Firebase/Auth/Source/SystemService/FIRAuthStoredUserManager.m

```diff
@@ -82,7 +82,12 @@ - (FIRUser *)getStoredUserForAccessGroup:(NSString *)accessGroup
   query[(__bridge id)kSecAttrAccount] = kSharedKeychainAccountValue;
 
   NSData *data = [self.keychainServices getItemWithQuery:query error:outError];
+#if !TARGET_OS_WATCH
   NSKeyedUnarchiver *unarchiver = [[NSKeyedUnarchiver alloc] initForReadingWithData:data];
+#else
+  NSError *error;
```

Comment:
Do you know what sort of errors are potentially expected here, and where we should be checking it?

Reply:
We should refactor it to using [GULSecureCoding](https://github.com/firebase/firebase-ios-sdk/blob/master/GoogleUtilities/Environment/Public/GULSecureCoding.h) eventually (may be a part of a follow up PR). No existing data migration will be required because the archive file format is the same for the secure coding.

Reply:
I didn't do anything to this. According to the documentation:
> This method throws an error if data isn’t a valid keyed archive.
while the deprecated method throws an exception if data is not a valid archive.